it/tutorials/scala-for-java-programmers.md

```diff
@@ -0,0 +1,716 @@
+esistono le classi equivalenti nella libreria delle classi di Scala--possiamo
+semplicemente importare le classi dei corrispondenti package Java:
+
+    import java.util.{Date, Locale}
+    import java.text.DateFormat
+    import java.text.DateFormat._
+
+    object FrenchDate {
+      def main(args: Array[String]) {
+        val now = new Date
+        val df = getDateInstance(LONG, Locale.FRANCE)
+        println(df format now)
+      }
+    }
+
+L’istruzione import di Scala è molto simile all’equivalente in Java
+tuttavia, risulta essere più potente. Più classi possono essere importate
+dallo stesso package includendole in parentesi graffe come nella prima riga
+di codice precedentemente riportato. Un’altra differenza è evidente
+nell’uso del carattere underscore (`_`) al posto dell’asterisco (`*`) per
+importare tutti i nomi di un package o di una classe. Questo perché
+l’asterisco è un identificatore valido (e.g. nome di un metodo), come
+vedremo più avanti.
+
+Inoltre, l’istruzione import sulla terza riga importa tutti i membri
+della classe `DateFormat`. Questo rende disponibili il metodo statico
+`getDateInstance` ed il campo statico `LONG`.
+
+All’interno del metodo `main` creiamo un’istanza della classe `Date` di
+Java che di default contiene la data corrente. Successivamente, definiamo il
+formato della data usando il metodo statico `getDateInstance` importato
+precedentemente. Infine, stampiamo la data corrente, formattata secondo la
+localizzazione scelta, con l’istanza `DateFormat`; quest’ultima linea mostra
+un’importante proprietà di Scala.I metodi che prendono un argomento possono
+essere usati con una sintassi non fissa. Questa forma dell’espressione
+
+    df format now
+
+è solo un altro modo meno esteso di scriverla come
+
+    df.format(now)
+
+Apparentemente sembra un piccolo dettaglio sintattico ma, presenta delle
+importanti conseguenze. Una di queste sarà esplorata nella prossima
+sezione.
+
+A questo punto, riguardo l’integrazione con Java, abbiamo notato che è
+altresì possibile ereditare dalle classi Java ed implementare le interfacce
+direttamente in Scala.
+
+
+## Tutto è un Oggetto
+
+Scala è un linguaggio orientato agli oggetti (_object-oriented_) puro nel
+senso che *ogni cosa* è un oggetto, inclusi i numeri e le funzioni. In questo
+differisce da Java che invece distingue tra tipi primitivi (come `boolean`
+  e `int` ) e tipi referenziati. Inoltre, Java non permette la manipolazione
+  di funzioni come fossero valori.
+
+### I numeri sono oggetti
+
+Poichè i numeri sono oggetti, hanno dei metodi. Di fatti
+un’espressione aritmetica come la seguente:
+
+    1 + 2 * 3 / x
+
+consiste esclusivamente di chiamate a metodi e risulta equivalente alla
+seguente espressione, come visto nella sezione precedente:
+
+    (1).+(((2).*(3))./(x))
+
+Questo significa anche che `+`, `*`, etc. sono identificatori validi in
+in Scala.
+
+Le parentesi intorno ai numeri nella seconda versione sono necessarie
```

Comment:
This is not true, at least not in Scala 2.11.7. For instance, the following is perfectly fine `1.+((2.*(3))./(x))`.